src/ocean/util/encode/Base64.d

```diff
@@ -503,10 +508,38 @@ unittest
     // decodeTest
     {
         ubyte[1024] decoded;
-        ubyte[] result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw==", decoded);
+        Const!(ubyte)[] result = decode("SGVsbG8sIGhvdyBhcmUgeW91IHRvZGF5Pw==", decoded);
```

Comment:
I'd just like to have a test that ensure we're getting a mutable buffer. Would it be possible to use another variable instead ? Actually it could be its own unittest block